django/db/models/fields/utils.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+
+class FieldCacheMixin(object):
+    """
+    Provides a common API for working with the fields
+    value cache on an instance.
+    """
+
+    def get_cache_name(self):
```

Comment:
This was added mostly to keep backwards compatibility and also because ForeignObjectRel used get_accessor_name() but as you pointed out on the comment below, I'm not overriding the get_cache_name(), looks like a bug on my side and also lack of coverage there. Will investigate further.